translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"retrieve via the\n"
+"\t\tcommand line tools. It will open the editor defined by your KUBE_EDITOR, "
+"or EDITOR\n"
+"\t\tenvironment variables, or fall back to 'vi' for Linux or 'notepad' for "
+"Windows.\n"
+"\t\tYou can edit multiple objects, although changes are applied one at a "
+"time. The command\n"
+"\t\taccepts filenames as well as command line arguments, although the files "
+"you point to must\n"
+"\t\tbe previously saved versions of resources.\n"
+"\n"
+"\t\tEditing is done with the API version used to fetch the resource.\n"
+"\t\tTo edit using a specific API version, fully-qualify the resource, "
+"version, and group.\n"
+"\n"
+"\t\tThe default format is YAML. To edit in JSON, specify \"-o json\".\n"
+"\n"
+"\t\tThe flag --windows-line-endings can be used to force Windows line "
+"endings,\n"
+"\t\totherwise the default for your operating system will be used.\n"
+"\n"
+"\t\tIn the event an error occurs while updating, a temporary file will be "
+"created on disk\n"
+"\t\tthat contains your unapplied changes. The most common error when "
+"updating a resource\n"
+"\t\tis another editor changing the resource on the server. When this occurs, "
+"you will have\n"
+"\t\tto apply your changes to the newer version of the resource, or update "
+"your temporary\n"
+"\t\tsaved copy to include the latest resource version."
+msgstr ""
+"\n"
+"\t\tEdit a resource from the default editor.\n"
+"\n"
+"\t\tThe edit command allows you to directly edit any API resource you can "
+"retrieve via the\n"
+"\t\tcommand line tools. It will open the editor defined by your KUBE_EDITOR, "
+"or EDITOR\n"
+"\t\tenvironment variables, or fall back to ‘vi’ for Linux or ‘notepad’ for "
+"Windows.\n"
+"\t\tYou can edit multiple objects, although changes are applied one at a "
+"time. The command\n"
+"\t\taccepts filenames as well as command line arguments, although the files "
+"you point to must\n"
+"\t\tbe previously saved versions of resources.\n"
+"\n"
+"\t\tEditing is done with the API version used to fetch the resource.\n"
+"\t\tTo edit using a specific API version, fully-qualify the resource, "
+"version, and group.\n"
+"\n"
+"\t\tThe default format is YAML. To edit in JSON, specify “-o json”.\n"
+"\n"
+"\t\tThe flag —windows-line-endings can be used to force Windows line "
+"endings,\n"
+"\t\totherwise the default for your operating system will be used.\n"
+"\n"
+"\t\tIn the event an error occurs while updating, a temporary file will be "
+"created on disk\n"
+"\t\tthat contains your unapplied changes. The most common error when "
+"updating a resource\n"
+"\t\tis another editor changing the resource on the server. When this occurs, "
+"you will have\n"
+"\t\tto apply your changes to the newer version of the resource, or update "
+"your temporary\n"
+"\t\tsaved copy to include the latest resource version."
+
+#: pkg/kubectl/cmd/drain.go:115
+msgid ""
+"\n"
+"\t\tMark node as schedulable."
+msgstr ""
+"\n"
+"\t\tMarca o node como agendável."
```

Comment:
Same as the drain above, "agendável" sounds strange, maybe "Remove a restrição de execução de workloads no node"